pkgs/blast_repo/lib/src/tweaks/github_action_tweak.dart

```diff
@@ -27,6 +27,11 @@ class GitHubActionTweak extends RepoTweak {
               'by SHA',
         );
 
+  @override
+  bool shouldRunByDefault(Directory checkout, String repoSlug) {
+    return true;
```

Comment:
I like `=>` for short things...but no biggy

Reply:
👍